sql/src/test/java/io/crate/integrationtests/FetchOperationIntegrationTest.java

```diff
@@ -392,7 +399,8 @@ public void onFailure(Throwable t) {
         latch.await();
 
         assertThat(resultingRows.size(), is(13));
-        assertThat(resultingRows.get(0).size(), is(outputSymbols.size()));
-        // TODO: validate sorting when CollectNode supports sorting
+        assertThat(resultingRows.get(0).length, is(outputSymbols.size()));
+        assertThat((Integer)resultingRows.get(0)[0], is(1));
```

Comment:
you could use TestingHelpers.printedTable(new CollectionBucket(...)); if you want to compare all the results